mmv1/templates/terraform/examples/network_security_client_tls_policy_advanced.tf.erb

```diff
@@ -0,0 +1,19 @@
+resource "google_network_security_client_tls_policy" "<%= ctx[:primary_resource_id] %>" {
+  provider               = google-beta
+  name                   = "<%= ctx[:vars]['resource_name'] %>"
+  labels                 = {
+    foo = "bar"
+  }
+  description            = "my description"
+  client_certificate {
+    certificate_provider_instance {
+        plugin_instance = "google_cloud_private_spiffe"
+      }
+    }
+  server_validation_ca {
```

Comment:
As serverValidationCa is an array, can you please add more server_validation_ca blocks here? Thanks.

Reply:
done